src/cmake/thirdparty/FindSCR.cmake

```diff
@@ -27,13 +27,30 @@ find_path( SCR_INCLUDE_DIRS scr.h
            NO_CMAKE_SYSTEM_PATH)
 
 find_library( SCR_LIBRARY NAMES scr libscr
```

Comment:
These can use `blt_find_libraries`:  https://llnl-blt.readthedocs.io/en/develop/api/utility.html#blt-find-libraries

It essentially has the same implementation but with some improved error handling:
https://github.com/LLNL/blt/blob/aea5fbf046e122bd72888dad0a7f97a07b9ff08d/cmake/BLTMacros.cmake#L1084-L1098

Reply:
It looks like the following can replace all of these calls
```cmake
blt_find_libraries(
        FOUND_LIBS SCR_LIBRARIES
        NAMES      scr jvtree dtcmp
        REQUIRED   TRUE
        PATHS      ${SCR_DIR}/lib64 ${SCR_DIR}/lib )
include(FindPackageHandleStandardArgs)
```